src/main/java/heronarts/lx/midi/surface/APC40Mk2.java

```diff
@@ -834,6 +975,30 @@ private void sendRainbowPickerGrid() {
     }
   }
 
+  // For each knob controlling a LinkedColorParameter, update its lighting to match
+  // the value it would change if turned right now. If we ever have an
+  // AggregateParameter knob where some subparameters are unipolar and others
+  // bipolar, we'll need to do a sendControlChange() as well here. For now,
+  // though, that never happens.
+  private void updateLCPKnobs() {
+    for (int i = 0; i < this.deviceListener.knobs.length; i++) {
+      LXListenableParameter knob = this.deviceListener.knobs[i];
+      if (knob instanceof LinkedColorParameter) {
```

Comment:
Can also handle basic `ColorParameter` here. I think method name of `updateColorKnobs()` would help for clarity.